core/node/state_keeper/src/health.rs

```diff
@@ -0,0 +1,30 @@
+use serde::{Deserialize, Serialize};
+use zksync_health_check::{Health, HealthStatus};
+use zksync_types::{L1BatchNumber, L2BlockNumber, H256};
+
+use crate::io::IoCursor;
+
+#[derive(Debug, Serialize, Deserialize)]
+pub struct StateKeeperHealthDetails {
+    pub next_l2_block: L2BlockNumber,
+    pub prev_l2_block_hash: H256,
+    pub prev_l2_block_timestamp: u64,
```

Comment:
Bikeshedding: Do these 2 fields provide valuable info? IMO, it'd make more sense to provide info like the number of txs in the current block.

Reply:
> the number of txs in the current block

I don't feel this info is somehow interesting. We have normally 2-3 tps  so we have to update it pretty frequent